### DISCUSSION TITLE
Open WebUI LXC added to Proxmox Helper-Scripts

### DISCUSSION THREAD

**Discussion by havardthom** [open-webui/open-webui#6513] [Show and tell]
Open WebUI LXC script with optional Ollama install has been added to Proxmox Helper-Scripts: https://tteck.github.io/Proxmox/#open-webui-lxc

Alternatively there is also a separate Ollama LXC script which builds Ollama from source and supports Intel iGPU passthrough (very long install time): `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ollama.sh)"`

**Comment by kylepyke** [comment]
> Alternatively there is also a separate Ollama LXC script which builds Ollama from source and supports Intel iGPU passthrough (very long install time): `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ollama.sh)"`

@havardthom This is fantastic, however, when I build the LXC, running ollama CLI commands return `command not found`. I can see that the server is running as a process, and going to the ip:11434 brings up a message which indicates it's ollama is running, but if I can't interact with it via the command line, how do I add models and run different models?

**Reply by Mariuss811** [reply]
Same issue. 
Did you find a fix ?

**Reply by Mariuss811** [reply]
I added `export PATH=$PATH:/opt/ollama` to `~/.bashrc` and reloaded it with `source ~/.bashrc`

**Comment by Nikishek** [comment]
I tried to see if Intel iGPU works for Open WebUI LXC, but it looks like it's still not supported? I'm using i3-N300 processor.
